build.gradle

```diff
@@ -95,13 +94,14 @@ project(":service") {
         baseName = rootProject.name
         classifier = null
         version = null
+        mainClassName = project.mainClassName
```

Comment:
@andrii-kovalenko-ct this is not needed, no?

Reply:
actually, it is not needed any more in section below (`manifest`): ad7c44c1

[Integrating with Application Plugin](http://imperceptiblethoughts.com/shadow/#integrating_with_application_plugin)